src/rpc/blockchain.cpp

```diff
@@ -1233,7 +1233,8 @@ UniValue z_gettreestate(const UniValue& params, bool fHelp)
             "\nResult:\n"
             "{\n"
             "  \"hash\": \"hash\",         (string) hex block hash\n"
-            "  \"height\": n,            (numeric) block height\n"
+            "  \"height\": n,              (numeric) block height\n"
+            "  \"time\": n,                (numeric) block time\n"
```

Comment:
Please specify the format and epoch.

Reply:
The PR got merged already, should I open another one?

Reply:
Sorry about that. If you don't mind, another PR would be great.

Reply:
Sure, the new one is here https://github.com/zcash/zcash/pull/5916.

Reply:
Resolved in https://github.com/zcash/zcash/pull/5916.